chains/tendermint/light.go

```diff
@@ -114,6 +115,13 @@ func (pr *Prover) LightClientWithoutTrust(db dbm.DB) (*light.Client, error) {
 		case h.GetRevisionHeight() == 0:
 			return fmt.Errorf("shouldn't be here")
 		default:
+			t, err := pr.chain.Timestamp(h)
+			if err != nil {
+				return err
+			}
+			if time.Now().Unix()-t.Unix() > int64(pr.getTrustingPeriod().Seconds()) {
```

Comment:
Don't use integer operations, but use the methods of `time.Time` and `time.Duration`.
Also, please note that in rare cases `time.Now()` can be less than `t`.

Reply:
Fixed
https://github.com/hyperledger-labs/yui-relayer/pull/103/commits/e356c01007bb6e10fedd483e74388f432cd6470c